test/metabase/query_processor/streaming/test_util.clj

```diff
@@ -0,0 +1,107 @@
+(ns metabase.query-processor.streaming.test-util
+  "Utility functions for testing QP streaming (download) functionality."
+  (:require [cheshire.core :as json]
+            [clojure.data.csv :as csv]
+            [clojure.test :refer :all]
+            [dk.ative.docjure.spreadsheet :as spreadsheet]
+            [metabase.query-processor :as qp]
+            [metabase.query-processor.streaming :as qp.streaming]
+            [metabase.test :as mt])
+  (:import [java.io BufferedInputStream BufferedOutputStream ByteArrayInputStream ByteArrayOutputStream InputStream InputStreamReader]))
```

Comment:
The stuff in this namespace used to be in `metabase.query-processor.streaming-test`, but I'm reusing some of it for testing exports in `metabase.api.dashboard-test`. That caused a circular reference, so I moved it into its own namespace.